### ISSUE TITLE
EPERM: operation not permitted

### ISSUE THREAD

**Issue by tomaschyly** [npm/write-file-atomic#49]
Since updating to **v3** I get this error:

EPERM: operation not permitted, rename 'C:\Users\Tomas\AppData\Roaming\MiArt\var\config\default.json.727851267' -> 'C:\Users\Tomas\AppData\Roaming\MiArt\var\config\default.json'

I use write-file-atomic to save app config inside **Electron v5 app**. This error only happens on **Windows**, on **MacOS** everything works fine. Also if I downgrade back to **2.4.3**, it works on **Windows** again.

Anything I can do to help solve this issue? Or is it something on my end?

**Comment by develar** [comment]
To disable, just pass `{chown: false, mode: false}`. I am going to send PR to `conf` module.

**Comment by tomaschyly** [comment]
I have checked my case again. And I noticed, that first time I can edit the file, but any subsequent edits show that error. It seems like they do not release control of the file after they are done editing.
Not sure and do not have the time to check it myself, I have too much other work to deal with.
Does the team not check issues here on github?

**Comment by tomaschyly** [comment]
I have recently tried to use the package again, now latest **v3.0.1**.
It is possible that they fixed it there, or other updates have dealt with this issue, but for me it seems to be working on multiple devices now. I have since then updated everything, including Electron to **v7**.

Since there is still no response, I will close this now. To others with this issue, try to test it again, maybe it will work now for you too.
